test/run_pass/require_main/foo.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = function() {
+  return require.main === module;
+}
```

Comment:
This file is not testing anything, move this to fixtures directory.

Reply:
But I have move it into `run_pass/require_main` dir, it is fault?

Reply:
Misunderstood, but a suggestion is the name to be require4 or add your test to exists require dir.

Reply:
Maybe it is more appropriate, I will do it.

Reply:
Thanks, the Node.js team maintains a huge tests now, ShadowNode doesn't want to do like that, thus we should own a better group for our tests.

Reply:
yeah, it is necessary！:)